src/managementgroups/azext_managementgroups/commands.py

```diff
@@ -0,0 +1,51 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import CliCommandType
+from ._client_factory import management_groups_client_factory, management_group_subscriptions_client_factory
+from ._exception_handler import managementgroups_exception_handler
+
+
+def load_command_table(self, _):
+    def managementgroups_type(*args, **kwargs):
```

Comment:
This is redundant now so remove the method and you can just use `CliCommandType` directly.

Reply:
Nice catch. Removed. Thanks!